src/main.rs

```diff
@@ -258,7 +263,21 @@ fn create_app<'a, 'b>(
                 .takes_value(true)
                 .possible_values(&["true", "false"])
                 .default_value("true"),
-        )
+        );
+
+    #[cfg(target_arch = "x86_64")]
+    {
+        app = app.arg(
```

Comment:
Doesn't it work to just put this up with the other `.arg(..)` so you can then get rid of the mut.

Reply:
Let me try!

Reply:
No luck with this, Rust does not like when I write:
```rust
        .arg(
            Arg::with_name("seccomp")
                .long("seccomp")
                .takes_value(true)
                .possible_values(&["true", "false"])
                .default_value("true"),
        )
        #[cfg(target_arch = "x86_64")]
        {
            .arg(
                Arg::with_name("sgx-epc")
                    .long("sgx-epc")
                    .help(config::SgxEpcConfig::SYNTAX)
                    .takes_value(true)
                    .min_values(1)
                    .group("vm-config"),
            )
        }
```
```
sboeuf@sboeuf:~/rust/cloud-hypervisor$ cargo build
   Compiling cloud-hypervisor v0.8.0 (/home/sebastien/rust/cloud-hypervisor)
error: expected one of `.`, `;`, `?`, `}`, or an operator, found `#`
   --> src/main.rs:262:9
    |
261 |         )
    |          - expected one of `.`, `;`, `?`, `}`, or an operator
262 |         #[cfg(target_arch = "x86_64")]
    |         ^ unexpected token
```

Reply:
:-(

Reply:
Did you try without the `{ .. }` ?

Reply:
Yes I did, same result, because the complaint is about the `#` from `#[cfg(target_arch = "x86_64")]` which is unexpected for the compiler.